backend/src/main/java/com/crm/exception/user/AccountAlreadyExistException.java

```diff
@@ -0,0 +1,8 @@
+package com.crm.exception.user;
+
+public class AccountAlreadyExistException extends CrmLoginException {
```

Comment:
This exception is handled as general CrmLoginException which returns a status code 400 (BAD REQUEST). I think better idea would be to return 409 (CONFLICT) in this case.